li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerConfig.java

```diff
@@ -39,6 +39,7 @@
   public static final String VALUE_DESERIALIZER_CLASS_CONFIG = ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG;
   public static final String SEGMENT_DESERIALIZER_CLASS_CONFIG = "segment.deserializer.class";
   public static final String AUDITOR_CLASS_CONFIG = "auditor.class";
+  public static final String ENABLE_PASSTHROUGH_CONFIG = ConsumerConfig.ENABLE_SHALLOW_ITERATOR_CONFIG;
```

Comment:
Why introduce a new config variable in `LiKafkaConsumerConfig` when the value is same as a vanilla config key? 
Also, it doesn't look like you are using this in `LiKafkaConsumerImpl`? Am I missing something?
Same question on `ENABLE_AUTO_COMMIT_CONFIG` , `AUTO_COMMIT_INTERVAL_MS_CONFIG`, `AUTO_OFFSET_RESET_CONFIG` and`MAX_POLL_RECORDS_CONFIG`

Reply:
discussed offline, just adding the enable.shallow.iterator config to the config constructor is sufficient.